docs/tutorial-run-hello-world.md

```diff
@@ -1,3 +1,11 @@
 # Running `hello-world.wasm` with Wasmtime
 
-... more coming soon
+Now you have built the `hello-world.wasm` binary (If you haven't,
+the instructions for doing so are in the
+[previous section](https://bytecodealliance.github.io/wasmtime/tutorial-create-hello-world.html)).
+
+You can then run the binary with Wasmtime, like so:
```

Comment:
Could this link to the documentation about how to install the CLI?

Could this section also be expanded with some words about how there's a number of ways to run wasmtime, and the CLI is only one example? This'd be a good spot to link to the other embeddings/languages too to showcase how to use wasmtime in other contexts.